tests/robot/libraries/rest_api.robot

```diff
@@ -7,6 +7,7 @@ Library        rest_api.py
 
 rest_api: Get
     [Arguments]      ${node}    ${uri}    ${expected_code}=200
+    Create Session   ${node}  http://localhost:9191
```

Comment:
I'm not sure this is correct. The agent is in a container and even though the ports are exported to host, they are different for each agent.

Did you have some issue without having this here?